prov/verbs/src/windows/verbs_nd_ibv.c

```diff
@@ -791,11 +783,25 @@ int ibv_post_recv(struct ibv_qp *qp, struct ibv_recv_wr *wr,
 	return 0;
 }
 
+static uint32_t check_for_inline(struct ibv_send_wr *wr, struct nd_qp *qp_nd)
+{
+	uint32_t length = 0;
+
+	for (int i = 0; i < wr->num_sge; ++i) {
+		length += wr->sg_list[i].length;
+	}
+
+	if (length <= qp_nd->max_inline_data) {
+		return ND_OP_FLAG_INLINE;
+	}
+
+	return 0;
+}
```

Comment:
This function shouldn't be needed.  The libfabric API returns the max_inject_size, which aligns with the max_inline_data size for verbs.  Callers should have already checked their message sizes and used either an inject call or the FI_INJECT flag to leverage this feature.  Adding this call may duplicate checks that were already made.